pages/converting.mdx

```diff
@@ -1,21 +1,25 @@
-# Converting to Qbox from Qb
+# Converting from QBCore
 
-1. Check your job grades in `qbx_core/shared/jobs.lua`. In Qbox, job grades are numbers without quotations, whereas in Qb they are strings.
-2. Configure ox_inventory and convert your database. Back it up first! https://overextended.dev/ox_inventory/Frameworks/qb
-3. Qbox has multicharacter built-in to core. If you want to keep using your own multicharacter resource, disable core's via core config. Otherwise delete your multicharacter resource.
-4. Qbox maintains a qb compatibility layer, so you can continue to use exports['qb-core'] and it should have full compatibibility. However, this means you won't get access to all the Qbox features and functions. So consider switching off of exports['qb-core'] when you can. This bridge layer means that qb-core resources should be 100% compatible with Qbox.
+1. Check your job grades in `qbx_core/shared/jobs.lua`. In Qbox, job grades are numbers without quotations, whereas in QBCore they are strings.
+2. Configure ox_inventory and convert your database. Back it up first! ([see ox_inventory documentation](https://overextended.dev/ox_inventory/Frameworks/qb))
+3. Qbox has multicharacter built into qbx_core. If you want to keep using your own multicharacter resource, configure qbx_core accordingly. Otherwise you can disable or delete your old multicharacter resource.
+4. Qbox has a queue system built into qbx_core. If you want to keep using your own queue system, specify `set qbx:enablequeue "true"` inside your cfg file. Otherwise you can disable or delete your old queue system.
+5. Qbox maintains a qb-core compatibility layer so all existing qb-core resources should expect 100% compatibility. You can continue using `exports['qb-core']`, however, this means you won't get access to all the new Qbox features and functions, so consider switching off of `exports['qb-core']` when you can.
```

Comment:
Maybe best to update language that 99% of resources built to run on qb-core should work since we do know of a few exceptions.

Reply:
Improved wording and added notes with 2402bd7.